GliaWidgets/View/Chat/Upgrade/ChatCallUpgradeStyle.swift

```diff
@@ -1,15 +1,42 @@
 import UIKit
 
+/// Style of the call upgrade view. Shown after engagement is upgraded to a call.
```

Comment:
This is a good comment IMO, not only says what it is but where the style is used. That is the kind of information that would be nice to have in other places as well.